devel/api.md

```diff
@@ -656,6 +663,46 @@ will also change the interfaces that are visible in the Nmstate state. Reducing
 the total VFs without setting `Interface.STATE` to `InterfaceState.ABSENT` for
 the corresponding interfaces might lead to an undefined system state or error.
 
+### `Ethernet.SRIOV.VFS_SUBTREE`
+
+If `Ethernet.SRIOV.TOTAL_VFS` is 1 or greater, the `Ethernet.SRIOV.VFS_SUBTREE`
+dictionary list allows to configure the VFS special options. Nmstate supports the
+following VF options:
+
+#### `Ethernet.SRIOV.VFS.ID`
+
+Type: `integer`
+
+ID of the VF to configure.
+
+#### `Ethernet.SRIOV.VFS.MAC_ADDRESS`
+
+Type: `string` in the format of `11:22:33:44:55:66`. Case insensitive.
```

Comment:
This is actually missing an explanation of the meaning of setting, it only explains the syntax. ;-)